files/zh-cn/mdn/writing_guidelines/page_structures/syntax_sections/index.md

```diff
@@ -0,0 +1,310 @@
+new IntersectionObserver(callback, options)
+```
+
+或这样（节选自 {{DOMxRef("Document.hasStorageAccess()")}} 页）：
+
+```js-nolint
+hasStorageAccess()
+```
+
+如果是静态方法，如 {{DOMxRef("URL.createObjectURL()")}}，则一同提供它的接口：
+
+```js-nolint
+URL.createObjectURL(object)
+```
+
+##### 多行/可选参数
+
+可用于多种不同方法的方法应扩展为多行，显示所有可能的变化。
+
+每个选项应独立成行，省略每个选项的注释和赋值。例如，{{jsxref("Array.prototype.slice()")}} 有两个可选参数，其文档如下所示：
+
+```js-nolint
+slice()
+slice(begin)
+slice(begin, end)
+```
+
+类似的示例是 {{DOMxRef("CanvasRenderingContext2D.drawImage")}}：
+
+```js-nolint
+drawImage(image, dx, dy)
+drawImage(image, dx, dy, dWidth, dHeight)
+drawImage(image, sx, sy, sWidth, sHeight, dx, dy, dWidth, dHeight)
+```
+
+{{jsxref("Date")}} 构造函数也是这样：
+
+```js-nolint
+new Date()
+new Date(value)
+new Date(dateString)
+new Date(year, monthIndex)
+new Date(year, monthIndex, day)
+new Date(year, monthIndex, day, hours)
+new Date(year, monthIndex, day, hours, minutes)
+new Date(year, monthIndex, day, hours, minutes, seconds, milliseconds)
+```
+
+##### 形式语法
+
+语法部分不应使用形式语法符号（使用[巴科斯范式](https://zh.wikipedia.org/wiki/巴科斯范式)），而应使用[如上所述](#多行可选参数)的扩展多行格式。
+
+虽然形式化符号为复杂语法的描述提供了一种简洁的机制，但它并不为许多开发人员所熟悉，而且可能与特定编程语言的有效语法相冲突。例如，“`[ ]`”既表示“可选参数”，也表示 JavaScript {{jsxref("Array")}} 。你可以从下面 {{jsxref("Array.prototype.slice()")}} 的形式语法中看到这一点：
+
+```js-nolint
+arr.slice([begin[, end]])
+```
+
+在认为有益的特定情况下，可使用形式通知声明单独的**形式语法**部分。
+
+##### 使语法块精确
+
+目的是使语法块尽可能纯粹、明确地定义功能的语法，不要包含任何无关的语法。例如，你可能会在网站的许多地方看到这种用于描述 promise 的语法形式：
+
+```js-nolint
+caches.match(request, options).then(function (response) {
+  // 使用 response 完成一些事情
+})
+```
+
+但下面这个版本要简洁得多，而且不包含多余的 {{JSxRef("Promise.prototype.then()")}} 方法调用：
+
+```js-nolint
+match(request, options)
+```
+
+##### 回调函数语法块
+
+对于接受回调函数的方法，要将该回调作为参数展示，而不是箭头函数或 `function` 表达式。
+
+```js-nolint
+filter(callbackFn)
+filter(callbackFn, thisArg)
+```
+
+然后，在“参数”章节，列出回调函数的参数和期望的返回类型。
+
+```md
+- `callbackFn`
+  - : 为数组中的每个元素执行的函数。它应该返回一个[真值](/zh-CN/docs/Glossary/Truthy)以将元素保留在结果数组中，否则返回一个[假值](/zh-CN/docs/Glossary/Falsy)。该函数被调用时将传入以下参数：
+    - `element`
+      - : 数组中当前正在处理的元素。
+    - `index`
+      - : 正在处理的元素在数组中的索引。
+    - `array`
+      - : 调用了 `filter()` 的数组本身。
+```
+
+##### 接受任意数量参数的语法
+
+对于接受任意数量参数的方法，语法块的写法如下：
+
+```js-nolint
+unshift()
+unshift(element1)
+unshift(element1, element2)
+unshift(element1, element2, /* …, */ elementN)
+```
+
+这里更倾向于从 1 开始编号，这样可以写出“`unshift` 在数组开头添加 N 个元素“这样的描述，也可以写成“第一个元素”（而不是“第零个元素”）。
+
+请注意，这里总是包含没有传递剩余参数的情况（即使没有意义）。然后，在“参数”章节中，像这样记录：
+
+```md
+- `element1`、…、`elementN`
+  - : 添加到 `arr` 开头的元素。
+```
+
+如果不传递剩余参数具有意义，请在这里添加 `\{{optional_inline}}`。
+
+另一个在剩余参数之前有一些位置参数的示例：
+
+```js-nolint
+splice(start)
+splice(start, deleteCount)
+splice(start, deleteCount, item1)
+splice(start, deleteCount, item1, item2)
+splice(start, deleteCount, item1, item2, /* …, */ itemN)
+```
+
+#### 参数章节
+
+然后，包含一个“参数”小节，其中以描述列表的方式解释了每个参数及其作用。如果参数是包含多个成员的对象，可以包含一个嵌套的描述列表，该列表本身也包含对每个成员的解释。可选参数应在其名称旁标注 \\{{optional_inline}} 宏调用。
+
+列表中每个参数的名称应包含在 {{HTMLElement("code")}} 块中。
```

Comment:
could we update this, as we are now using inline code rather than `<code>` in HTML

Reply:
PR filed mdn/content#29182